File: src/main/java/um/haberes/core/model/view/CargoLiquidacionNoDocente.java
package um.haberes.core.model.view;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import org.hibernate.annotations.Immutable;

@Entity
@Table(name = "vw_cargo_liquidacion_no_docente")
@Immutable
@Getter
@Setter
@EqualsAndHashCode
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class CargoLiquidacionNoDocente {

    @Id
    private String uniqueId = null;

    private Long legajoId = null;

    private Integer anho = null;

    private Integer mes = null;
}

File: src/main/java/um/haberes/core/kotlin/model/view/CargoLiquidacionNoDocente.kt
package um.haberes.core.kotlin.model.view

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table
import org.hibernate.annotations.Immutable

@Entity
@Table(name = "vw_cargo_liquidacion_no_docente")
@Immutable
data class CargoLiquidacionNoDocente(

    @Id
    var uniqueId: String? = null,

    var legajoId: Long? = null,
    var anho: Int? = null,
    var mes: Int? = null

)
